coderd/users.go

```diff
@@ -386,6 +386,20 @@ func (api *API) postUser(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(req.OrganizationIDs) == 0 {
+		httpapi.Write(ctx, rw, http.StatusBadRequest, codersdk.Response{
+			Message: "No organization specified to place the user as a member of. It is required to specify at least one organization id to place the user in.",
+			Detail:  "required at least 1 value for the array 'organization_ids'",
+			Validations: []codersdk.ValidationError{
+				{
+					Field:  "organization_ids",
+					Detail: "Missing values, this cannot be empty",
+				},
+			},
+		})
+		return
+	}
```

Comment:
To clarify, when creating a user they must be in at least 1 org, but later on it's possible to remove them and they have 0 orgs. What is the limitation requiring an org on creation? Just more UI work?

Reply:
This is correct. There is no limitation, just UX work. This was the existing constraint, and I did not want to change it yet. It'll have to change before multi-org goes live though.